### PR TITLE
Features/travis ci

### LINKED ISSUE #5: Tests and linting through Travis CI
At the moment:
- tests and linting are not run through a CI.
- tests are too few.
- there are multiple linting errors if you run `npm run eslint`

I suggest we use Travis CI to run tests and linting, in order to ease contribution and code quality.

@KillianKemps Would you like to work on this? 
Of course, I'm available to give you a hand if needed - not that I think you need help of any kind :)

### PR DESCRIPTION
**Pull request by KillianKemps**
Add .travis.yml to launch Travis CI continuous tests and linting.

Closes #5 

:warning: Travis CI must be plugged to this repository before merge.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+  - "4.1"
+  - "4.0"
+  - "0.12"
+  - "0.11"
+  - "0.10"
+  - "iojs"
```
Comment (ziir) [inline .travis.yml]: iojs is not in scope of supported versions, no need to pollute the builds with it in my opinion.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+  - "4.1"
+  - "4.0"
+  - "0.12"
+  - "0.11"
```
Comment (ziir) [inline .travis.yml]: 0.11 is an unstable version, we don't need to test for an unstable version in my opinion.

**Comment by ziir** [comment]
Hey @KillianKemps, thanks for working on this \o/

File: .travis.yml (hunk 2 above)
Comment (KillianKemps) [inline .travis.yml]: Allright, I removed it as well as iojs.

**Comment by ziir** [comment]
I have enabled Travis CI on the repo.
It would be great to add the build-status icon in our Readme, feel free to PR it if you want :)

:+1:

**Pull request merged**